src/app/projects/[project_id]/page.tsx

```diff
@@ -0,0 +1,172 @@
+        <p>{description}</p>
+        <Button
+          className="join-us__button"
+          variant="contained"
+          size="large"
+          href="https://tally.so/embed/w4207A"
+        >
+          Join Us
+        </Button>
+      </div>
+      <div className="project-banner__image">
+        <Image
+          src={projectImage}
+          alt="CodeForBC banner image"
+          width={0}
+          height={0}
+        />
+      </div>
+    </section>
+  );
+}
+
+function ProjectOverview(projectData: ProjectInterface) {
+  const { projectOverview } = projectData;
+  if (!projectOverview) {
+    return null;
+  }
+
+  const { lookingFor, problemStatement, projectObjectives, projectLead } =
+    projectOverview;
+  return (
+    <section className="outer-box">
+      <div className="inner-box project-page-overview">
+        <h1>Project Overview</h1>
+        <div className="descriptor">
+          <h2>Problem Statement</h2>
+          <p>{problemStatement}</p>
+        </div>
+        <div className="descriptor">
+          <h2>Project Objectives</h2>
+          <ul className="project-objectives__list">
+            {projectObjectives.map((objective: string, idx: number) => (
+              <li key={`objective-key-${idx}`}>{objective}</li>
+            ))}
+          </ul>
+        </div>
+        <div className="descriptor">
+          <h2>Project Lead</h2>
+          <p>{projectLead}</p>
+        </div>
+        {lookingFor.length > 0 && (
+          <>
+            <h2>We Are Looking For</h2>
+            <div className="looking-for-container">
+              {lookingFor.map((item: string, idx: number) => (
+                <span
+                  className="looking-for-item"
+                  key={`looking-for-item-${idx}`}
+                >
+                  {item}
+                </span>
+              ))}
+            </div>
+          </>
+        )}
+      </div>
+    </section>
+  );
+}
+
+function Technologies(projectData: ProjectInterface) {
+  const { languages, links, tools } = projectData;
+
+  return (
+    <section className="outer-box">
+      <div className="inner-box">
+        <h1>Technologies Used</h1>
+        <div className="descriptor">
+          <h2>Languages</h2>
+          <p>{languages?.join(', ')}</p>
+        </div>
+        <div className="descriptor">
+          <h2>Tools</h2>
+          <p>{tools?.join(', ')}</p>
+        </div>
+        <div className="descriptor">
+          <h2>Links</h2>
+          {links.map((link: LinkInterface, idx: number) => (
+            <Link href={link.url} key={`project-link-${idx}`}>
+              <Image
+                src="/assets/github-mark.svg"
+                alt={link.title}
+                width={20}
+                height={20}
+              />
+            </Link>
+          ))}
+        </div>
+      </div>
+    </section>
+  );
+}
+
+function ProposeProject() {
+  return (
+    <section className="propose-project">
+      <hr />
```

Comment:
there is a responsiveness issue with mobile screen size
![pr review](https://github.com/CodeForBc/codeforbc-web/assets/105146344/e7f19c04-d89b-48e7-abc2-e6b302218e60)

Reply:
It's interesting, I can reproduce this when I actively change my browser's size from Desktop -> Mobile

However, if you navigate to these pages without transitioning (i.e. start at Mobile size), then this doesn't happen:

https://github.com/CodeForBc/codeforbc-web/assets/16601729/b06ab803-56c4-4eb4-b122-c67f472b6d57

(Ignore the Vercel Toolbar)

Reply:
Timeboxed it to 20 minutes, managed to find the source of the issue.

Fixed in [8e0078e](https://github.com/CodeForBc/codeforbc-web/pull/50/commits/8e0078e447320bde8c3e4d6ae5ba1f9a0d57c091)